packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -837,6 +870,13 @@ class Tooltip extends OverlayClassMixin(ThemePropertyMixin(ElementMixin(PolymerE
       this.__oldContext = context;
     }
   }
+
+  /** @private */
+  __updateSrLabelText(srLabel, textContent) {
+    if (srLabel) {
+      srLabel.textContent = textContent;
```

Comment:
It seems the PR lacks a test to verify that the text content is correctly rendered in the sr label.

Reply:
Thanks, added tests for `text` and `generator` properties.